Interface/EmotionalCities.Pluma/EmotionalCities.Pluma.csproj

```diff
@@ -18,13 +18,13 @@
     <PackageOutputPath>..\bin\$(Configuration)</PackageOutputPath>
     <PackageTags>Bonsai Rx Emotional Cities Harp Pluma</PackageTags>
     <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
-    <VersionPrefix>0.1.0</VersionPrefix>
+    <VersionPrefix>0.1.1</VersionPrefix>
```

Comment:
The version number should be incremented to `0.2.0` since this adds new features to the interface.

Reply:
4. which version should be incremented? The one the one is now availabe "_as is_", or the one will be modified after this talk?

Reply:
Fix in 40357405ab167955d309dbc49de00bd0dcc0a6e1